packages/api/database/migrations/1603677784030_difficulties.ts

```diff
@@ -0,0 +1,18 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
```

Comment:
This migration will remove the difficulties table but foreign keys in `rounds` and `games` tables will still be there. We don't really have deploy the game in production, so we could maybe just delete `1587805740197_difficulties.ts` and foreign keys ?